Cargo.toml

```diff
@@ -31,19 +32,23 @@ repository = "https://github.com/worldcoin/orb-software"
 rust-version = "1.77.0" # See rust-version.toml
 
 [workspace.dependencies]
+alsa-sys = "0.3.1"
```

Comment:
`alsa-sys` is used only by the orb-ui/sound crate, and won't be used by another crate I think, we might want to move it to the Cargo.toml of that crate, what do you think? 🤔

Reply:
Sure. I don't actually have a strong opinion either way. We can do that.